contrib/hadoop-store-builder/src/java/voldemort/store/readonly/disk/HadoopStoreWriter.java

```diff
@@ -55,6 +55,27 @@
 
     private static final Logger logger = Logger.getLogger(HadoopStoreWriter.class);
 
+    /**
+     * For each chunk in a partition, Index files contains the integer offset into
+     * the corresponding data file. So it is capped at {@link Integer#MAX_VALUE}.
+     */
+    public static final long MAX_CHUNK_SIZE = Integer.MAX_VALUE;
+
+    /**
+     * The default chunk size tries to leave an error margin so that compressed
+     * data size on hadoop or skewed data sets don't cause chunk overflows
+     */
+    public static final long DEFAULT_CHUNK_SIZE = MAX_CHUNK_SIZE / 2;
```

Comment:
Is this 'DEFAULT_CHUNK_SIZE' being used in VoldemortBuildAndPushJob.java?
I saw it is still using '1024 * 1024 * 1024'.
https://github.com/voldemort/voldemort/blob/master/contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java#L744